File: PRAK 403/PRAK403_2310817110011_Ghani_Mudzakir.c
#include <stdio.h>

int main()
{
    int nilai1, nilai2;
    int nilai_batas;

    scanf("%d %d", &nilai1, &nilai2);
    nilai_batas = (nilai1-nilai2);    
    if (nilai_batas <0)
    {
        nilai_batas = nilai_batas*-1;
    for (int i = 0; i <= nilai_batas; i++)
    {
        int j = 1;

        printf("%d-%d ", nilai1, nilai2);

        nilai1 = nilai1+j;
        nilai2 = nilai2-j;

    }
    }
    else{
    for (int i = 0; i <= nilai_batas; i++)
        {
            int j = 1;

            printf("%d %d  ", nilai1, nilai2);

            nilai1 = nilai1-j;
            nilai2 = nilai2+j;
        }
   
    }
    

    return 0;
}


    

File: PRAK 403/PRAK403_2310817110011_Ghani-Mudzakir.py
nilai1, nilai2 = input().split()
nilai1 = int(nilai1)
nilai2 = int(nilai2)

nilai_batas = nilai1-nilai2

if nilai_batas<0:
    nilai_batas = nilai_batas*(-1)
    for x in range(-1,nilai_batas):
        i = 1
        print(f"{nilai1}-{nilai2} ", end=" ")
        nilai1 += i
        nilai2 -= i
elif nilai_batas>0:
    for x in range(-1,nilai_batas):
        i = 1
        print(f"{nilai1}-{nilai2} ", end=" ")
        nilai1 -= i
        nilai2 += i




